playbooks/roles/openvpn/vars/main.yml

```diff
@@ -24,3 +24,9 @@ openvpn_firewall_rules:
   - iptables -t nat -A POSTROUTING -s 10.9.0.0/24 -o {{ ansible_default_ipv4.interface }} -j MASQUERADE
 
 openvpn_gateway_location: "{{ streisand_gateway_location }}/openvpn"
+
+# If stunnel is enabled generating
```

Comment:
Apologies - meant to leave these two comments as part of a review but hit the wrong buttons in my substandard coffee haze. I'll leave them as-is for now